server/acl_casbin_policy_dba.csv

```diff
@@ -25,6 +25,10 @@ p, DBA, /project/{projectID}/webhook/{webhookID}/test, GET
 p, DBA, /environment, POST
 p, DBA, /environment, GET
 p, DBA, /environment/{id}, PATCH
+p, DBA, /policy, GET
+p, DBA, /policy/{policyID}, GET
```

Comment:
policyID -> id, to be consistent with others

We used environmentID below to differentiate the confusion between policy and environment ID.

And any reason we cannot use "/policy/environment/{environmentID}" APIs below?

Reply:
The `/policy/environment/{environmentID}` is the operation based by `environment` + `policy type` identifier. It will override the `payload` for a specific `environment` + `policy type` combination, or create a new one if the identifier not exist.

This means if users have the schema review policy like:

```json
{
  "id": "1",
  "name": "schema review for prod",
  "environment": "prod",
  "type": "bb.policy.schema-review",
  "payload": "1"
}
```

If users call that endpoint to update the first policy (`id=1`) payload to `3`:

```json
// request body
{
  "id": "1",
  "name": "schema review for prod",
  "environment": "dev", // user want to update the environment
  "type": "bb.policy.schema-review",
  "payload": "3" // user also want to update the payload at the same time
}
```

**Unexpected situation 1**

If the user has another policy (`id=2`):

```json
// another existing policy
{
  "id": "2",
  "name": "schema review for dev",
  "environment": "dev",
  "type": "bb.policy.schema-review",
  "payload": "2"
}
```

The final result could be:

```json
// We will update the wrong policy. The target policy `id=1` has not changed.
{
  "id": "1",
  "name": "schema review for prod",
  "environment": "prod",
  "type": "bb.policy.schema-review",
  "payload": "1"
}

{
  "id": "2",
  "name": "schema review for dev",
  "environment": "dev",
  "type": "bb.policy.schema-review",
  "payload": "3"
}
```

**Unexpected situation 2**

If the user doesn't have any other policies, the final result could be:


```json
// We will create a new policy. The target policy `id=1` has not changed.
{
  "id": "1",
  "name": "schema review for prod",
  "environment": "prod",
  "type": "bb.policy.schema-review",
  "payload": "1"
}

{
  "id": "2",
  "name": "schema review for prod",
  "environment": "dev",
  "type": "bb.policy.schema-review",
  "payload": "3"
}
```